src/node.cc

```diff
@@ -2786,24 +2789,22 @@ static void EnvQuery(Local<Name> property,
                      const PropertyCallbackInfo<Integer>& info) {
   int32_t rc = -1;  // Not found unless proven otherwise.
   if (property->IsString()) {
-#ifdef __POSIX__
     node::Utf8Value key(info.GetIsolate(), property);
-    if (getenv(*key))
-      rc = 0;
-#else  // _WIN32
-    node::TwoByteValue key(info.GetIsolate(), property);
-    WCHAR* key_ptr = reinterpret_cast<WCHAR*>(*key);
-    GetEnvironmentVariableW(key_ptr, nullptr, 0);
-    if (GetLastError() != ERROR_ENVVAR_NOT_FOUND) {
+    // We are only interested in exsistance, so we can keep the buffer small.
+    char buffer[256];
+    size_t = sizeof(buffer);
```

Comment:
Does this compile?

Reply:
Well obviously not (which means my way of locally compiling is broken. I was trying to not compile everything 'cause Windows)